src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/NumberBoxTests/MUX_Test.xaml

```diff
@@ -6,12 +6,11 @@
 			 xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
 			 xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 			 xmlns:contract5Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-			 xmlns:xamarin="http://uno.ui/xamarin"
-			 mc:Ignorable="d xamarin"
+			 mc:Ignorable="d"
 			 d:DesignHeight="300"
 			 d:DesignWidth="400">
 
-	<xamarin:Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+	<Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
```

Comment:
Won't this fail, since SamplesApp.UWP doesn't have WinUI 2 yet?

Reply:
Well it builds so apparently not :D

Reply:
I think it referenes the latest stable :-) .

Reply:
of course because it's using the Fluent themes 🤦... I think I was thinking of the toolkit...